src/mds/Locker.h

```diff
@@ -287,6 +287,8 @@ class Locker {
 
   void issue_client_lease(CDentry *dn, client_t client, bufferlist &bl, utime_t now, Session *session);
   void revoke_client_leases(SimpleLock *lock);
+  void encode_infinite_lease(bufferlist& bl, const Session* session);
+  void encode_null_lease(bufferlist& bl, const Session* session);
```

Comment:
API should be the same as for `static void CDir::encode_empty_dirstat(bufferlist& bl, const session_info_t& info)` mentioned in my other comment.